core/platforms/irix/bareos-dir.in

```diff
@@ -12,7 +12,7 @@ RETVAL=0
 case "$1" in
     start)
        echo "Starting the Bareos Director: "
-       @sbindir@/bareos-dir $2 -c @confdir@/bareos-dir.conf
+       @sbindir@/bareos-dir $2
```

Comment:
I think that IRIX also can be considered dead so we should remove it completely.

Reply:
Removed.